pkg/rpc/server/server.go

```diff
@@ -1154,36 +1157,42 @@ func (s *Server) invokeContractVerify(reqParams request.Params) (interface{}, *r
 		return nil, responseErr
 	}
 
-	args := reqParams[1:2]
-	var tx *transaction.Transaction
+	bw := io.NewBufBinWriter()
+	if len(reqParams) > 1 {
+		args := reqParams[1] // second `invokecontractverify` argument is an array of arguments for `verify` method
+		if len(args.Value.([]request.Param)) > 0 {
```

Comment:
Won't it panic if second argument is omitted?

Reply:
No, why should it panic? We also have a test for that.

Reply:
Wanted to write "if it is of wrong type". I see it is fixed now.